CombinatorialGames/Surreal/Cut.lean

```diff
@@ -0,0 +1,164 @@
+/-
+Copyright (c) 2025 Aaron Liu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Liu, Violeta Hernández Palacios
+-/
+import Mathlib.Order.UpperLower.CompleteLattice
+import CombinatorialGames.Mathlib.Concept
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Surreal cuts
+
+A surreal cut is defined as consisting of two sets of surreals with the following properties:
+
+- They are complementary sets.
+- Every surreal in the first set is less than every surreal in the second set.
+
+This construction resembles the construction of the surreals themselves, but yields a "bigger"
+structure, which can embed the surreals, but is also a complete linear order.
+
+Note that surreal cuts are is **not** the same as the Dedekind completion of the surreals. Whereas
+the Dedekind completion maps every element of the original order to a unique Dedekind cut, every
+surreal number `x` actually corresponds to two cuts `(Iio x, Ici x)` and `(Iic x, Ioi x)`, which we
+call the left and right cut, respectively.
+
+The theory of concept lattices gives us a very simple definition of surreal cuts as
+`Concept Surreal Surreal (⬝ < ⬝)`. However, we've attempted to provide a thin wrapper for all
+concept terminology.
+-/
+
+universe u
+
+namespace Surreal
+open Set IGame
+
+/-- A surreal cut consists of two complementary sets of surreals, where every surreal in the former
+is less than every surreal in the latter. -/
+abbrev Cut := Concept Surreal Surreal (· < ·)
+
+namespace Cut
+
+/-- The left set in a cut. This is an alias for `Concept.extent`. -/
+def left (x : Cut) := x.extent
```

Comment:
Should these be more reducible? (i.e. moving to `abbrev`?)

Reply:
I don't think so? I think that might interact poorly with `simp`, which might end up applying lemmas that rewrite `left`/`right` as `extent`/`intent`.